V2 Rewording the Manifesto (#7)
anttiviljami/product-engineer-manifesto: README.md

<div>&nbsp;</div>
</div>

It is our responsibility as builders to first seek to understand the problem, before
diving into solutions.

Our craft combines _Product Thinking_ with great technical execution.

We concern ourselves with design, technical, and business domains;
taking an active part in each to shape the product and empower others to do the same.

We take great professional pride in the products we ship and refuse to
limit ourselves to technical-only roles in our teams.

## The Product Engineer Mindset

In our work as _Product Engineers_, we have come to value:

- **Continuous delivery of working software** over promises and estimations
- **Asking why** to understand the customer problem before diving into code
- **Customer collaboration and feedback** over tickets and second-hand knowledge
- **Teamwork and communication** over picking up tasks and working in isolation

> - **Testing the product ourselves** over relying on others (especially our users) to find issues

Reviewer: ```suggestion
- **Testing the product ourselves over making users the testers of our work
```

Author: Hmm, I still prefer "relying on others to find issues". 

"Testers of our work" seems somehow clumsy to me.
